cluster-up/cluster/kind-k8s-sriov-1.17.0/manifests/multus.yaml

```diff
@@ -0,0 +1,147 @@
+---
```

Comment:
This file should be updated to the file you download [here](https://github.com/kubevirt/kubevirtci/pull/283/files?file-filters%5B%5D=.sh#diff-7f40e851da428ab9b276536729dfdac7R87).